hc/api/tests/test_admin.py

```diff
@@ -17,3 +20,6 @@ def test_it_shows_channel_list_with_pushbullet(self):
         ch.save()
 
         ### Assert for the push bullet
+        is_present = Channel.objects.filter(user=self.alice, kind='pushbullet', value="test-token").exists()
+        self.assertTrue(is_present, msg='pushbullet not present')
```

Comment:
No...it's checking whether the pushbullet is present. the message is displayed when the test fails.